zerver/openapi/zulip.yaml

```diff
@@ -1384,38 +1384,39 @@ paths:
                                     - delete
                                 streams:
                                   type: array
+                                  deprecated: true
                                   description: |
-                                    Array of objects, each containing
-                                    details about a channel that was deleted.
+                                    Array of objects, each containing stream_id of the channel
```

Comment:
"Array of objects, each containg ID of the channel that was deleted." would be better here.